Formula/thrift.rb

```diff
@@ -33,6 +33,11 @@ class Thrift < Formula
   depends_on "libevent" => :optional
   depends_on "python" => :optional
 
+  if build.with? "java"
+    depends_on :java => "1.8"
+    depends_on "ant" => :build
```

Comment:
feels like `=> :build` wants to go first

Reply:
Probably. I was kinda throwing stuff at the wall stylistically whilst trying to simply get this to build the right way. I'll tweak it a bit now it is building correctly.